src/js/modules/infragistics.ui.widget.js

```diff
@@ -98,36 +99,93 @@
 				return $.ig.regional.defaults[ key ];
 			}
 		},
+		_getLocaleWidgetName: function () {
+			this.localeWidgetName = this.localeWidgetName ||
```

Comment:
AFAIK it is recommended to try using widgetFullName(in older jQuery versions widgetName was only available)